### DISCUSSION TITLE
Is Saleor right choice for subscription based digital products?

### DISCUSSION THREAD

**Discussion by oybekalimat** [saleor/saleor#11057] [General]
Hey everyone, we are looking for a new e-commerce platform, and we can't decide whether Saleor is the right choice. Could you please share your thoughts? We currently have following requirements:

1. Does Saleor support subscription based digital products/service?
2. Is it possible to remove concepts like warehouse and stock management?
3. Is it possible to apply discount per product based on quantity in cart?
4. Is it possible to make product to belong to multiple categories?
5. Custom dashboard to manage clients' subscriptions, list customers, and show their current subscription status, when was their last payment, what plan they have chosen, and when payment is due. 

What do you think, is it possible to solve our requirements with custom plugin or with app without forking dashboard and without forking Saleor core and modifying code. 
 
Yes, there are hacky solution for each requirements, but with hacky solutions it is becoming more and more unclear, buggy, and difficult to explain to client (store owner). 

Thank you!

**Comment by peelar** [comment]
Hello @oybekalimat 👋 

I am afraid Saleor was not designed with digital products in mind. All my answers to your questions would vary from "no" to "not really", so you would have to invest some serious time to make it work. 

I hope that brings some clarity, have a nice day!

**Reply by oybekalimat** [reply]
Thank you for your response, I really appreciate it! Have a nice day!
